scripts/Tools/pelayout

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+
+"""
+This utility allows the CIME user to view and modify a case's PE layout.
```

Comment:
Perhaps you should not include the comment about modifying until that functionality is included.